src/main/perl/RuleBasedEditor.pm

```diff
@@ -148,27 +152,27 @@ end of the line if it is modified by L<CAF::RuleBasedEditor>.
 
 =item *
 
-LINE_FORMAT_KEY_VAL:        key val (e.g. Xrootd, Apache)
+LINE_FORMAT_KW_VAL:        key val (e.g. Xrootd, Apache)
```

Comment:
I found `KEY` misleading and had in mind that `KW` as `keyworkd` was probably better. I'm happy to revert it or change to something else if you don't agree...